components/templates/NewsPage/index.tsx

```diff
@@ -23,96 +24,159 @@ const sanitize = (dirty: string | undefined) => {
 
     return sanitizeHtml(dirty, sanitizeOptions);
   } catch {
-    return undefined;
+    return "";
   }
 };
 
+interface ContentBlockArgs {
+  text: string;
+  id: string;
+}
+
+const makeTextContentBlock = ({
```

Comment:
these helpers seek to start normalizing NOIRLab content with our own entries, by putting the text content of a NOIRLab entry into content blocks rather than having separate rendering with conditional logic